jupyterhub/apihandlers/users.py

```diff
@@ -18,6 +18,43 @@ def get(self):
         users = self.db.query(orm.User)
         data = [ self.user_model(u) for u in users ]
         self.write(json.dumps(data))
+    
+    @admin_only
+    @gen.coroutine
+    def post(self):
+        data = self.get_json_body()
+        print(data)
```

Comment:
was this meant to be for debugging temporarily? I'm going to guess real messages go to logging?